### ISSUE TITLE
snowflake_stage_grant docs have the wrong number of arguments

### ISSUE THREAD

**Issue by jtzero** [chanzuckerberg/terraform-provider-snowflake#444]
**Provider Version**
v0.23.2 

**Terraform Version**
v0.12.29

**Describe the bug**
The documentation states the following
```
# format is stage name | privilege | true/false for with_grant_option
terraform import snowflake_stage_grant.example 'stageName|USAGE|true'
```

and when doing that, the import fails with the following
```
❯ tf import snowflake_stage_grant.grant_my_stage 'MY_STAGE|OWNERSHIP|false'
snowflake_stage_grant.grant_my_stage: Importing from ID "MY_STAGE|OWNERSHIP|false"...
snowflake_stage_grant.grant_my_stage: Import prepared!
  Prepared snowflake_stage_grant for import
snowflake_stage_grant.grant_my_stage: Refreshing state... [id=MY_STAGE|OWNERSHIP|false]

Error: 4 or 5 fields allowed
```

**Expected behavior**
the documentation should be updated to something akin to the following
```
# format is databasename | schemaname | stage name | privilege | true/false for with_grant_option
terraform import snowflake_stage_grant.example 'database_name|schema|stageName|USAGE|true'
```

**Code samples and commands**

**Additional context**

**Comment by ryanking** [comment]
Thanks for catching this. You can fix this by updating [this file](https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/main/examples/resources/snowflake_stage_grant/import.sh) and running `make docs`. PR welcome.

**Comment by yohei1126** [comment]
This issue has been fixed by #429
